src/components/EventCard/EventCard.js

```diff
@@ -56,23 +56,23 @@ const EventCard = ({
         {startDate && (
           <div className="EventCard__event-info-wrapper">
             <div className="EventCard__icon EventCard__icon--calendar" />
-            <p className="EventCard__event-info-text">
+            <time className="EventCard__event-info-text">
               {getEventDate(startDate)}
               {endDate &&
                 !datesAreOnSameDay(startDate, endDate) &&
                 ` - ${getEventDate(endDate)}`}
-            </p>
+            </time>
           </div>
         )}
         {startDate && (
           <div className="EventCard__event-info-wrapper">
             <div className="EventCard__icon EventCard__icon--clock" />
-            <p className="EventCard__event-info-text">
+            <time className="EventCard__event-info-text">
```

Comment:
You used, the tag, but forgot main attribute `datetime`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time